test/src/core/lib/seer-integration.spec.js

```diff
@@ -10,17 +10,19 @@ test('Seer overrides', t => {
     one: 1
   };
 
-  window.__SEER_INITIALIZED__ = false;
+  if (isBrowser) {
```

Comment:
@balthazar FYI is this the right approach, to avoid seer tests completely under node (and thus under CI)?